packages/dds/tree/src/simple-tree/api/schemaFactory.ts

```diff
@@ -176,7 +176,7 @@ export type ScopedSchemaName<
  * As a workaround, we have this type as a third place which can be linked.
  * @system @sealed @public
  */
-export const schemaStatics = {
+export interface SchemaStatics {
```

Comment:
the approach of using a constant with doc comments on its members works poorly with API-Documenter (you can't link it and they don't show up in generated docs, only inherit doc works). This this switched to a separate interface and instance.